OpenAI-DotNet/Common/BaseResponse.cs

```diff
@@ -68,12 +69,86 @@ public abstract class BaseResponse
         [JsonIgnore]
         public string ResetRequests { get; internal set; }
 
+        /// <summary>
+        /// The time until the rate limit (based on requests) resets to its initial state represented as a TimeSpan.
+        /// </summary>
+        [JsonIgnore]
+        public TimeSpan ResetRequestsTimespan { get => ConvertTimestampToTimespan(ResetRequests); }
+
         /// <summary>
         /// The time until the rate limit (based on tokens) resets to its initial state.
         /// </summary>
         [JsonIgnore]
         public string ResetTokens { get; internal set; }
 
+        /// <summary>
+        /// The time until the rate limit (based on tokens) resets to its initial state represented as a TimeSpan.
+        /// </summary>
+        [JsonIgnore]
+        public TimeSpan ResetTokensTimespan { get => ConvertTimestampToTimespan(ResetTokens); }
```

Comment:
can probably be even simpler:
```suggestion
        public TimeSpan ResetTokensTimespan => ConvertTimestampToTimespan(ResetTokens);
```

Reply:
Interesting. I did not know that this syntax exists. I'll commit these changes in a moment.

Reply:
Change committed